### PR TITLE
3751 show coop

### PR DESCRIPTION
**Pull request by emitche**
Shows individual record for a coop

- Also includes #3754 edit coop

### CODE REVIEW

File: config/initializers/simple_form_bootstrap.rb [hunk 1]
```diff
@@ -0,0 +1,149 @@
+# Use this setup block to configure all options available in SimpleForm.
```
Comment (emitche) [inline config/initializers/simple_form_bootstrap.rb]: Simple Form errors can be tied into our existing alerts.

File: app/views/admin/organizations/new.html.slim [hunk 2]
```diff
@@ -0,0 +1,14 @@
+- content_for :styles
```
Comment (emitche) [inline app/views/admin/organizations/new.html.slim]: Do we need a form to create new coops?

File: app/views/admin/organizations/index.html.slim [hunk 3]
```diff
@@ -3,17 +3,19 @@
...
-  - g.column name: 'ID', attribute: 'id'
+  - g.column name: 'ID', attribute: 'id' do |organization|
+    - link_to(organization.id, admin_organization_path(organization.id))
 
   - g.column name: 'Name', attribute: 'name'
 
   - g.column name: 'City', attribute: 'city'
 
-  - g.column name: 'Country' do |organization|
+  - g.column name: 'Country', attribute: 'name', model: Country do |organization|
```
Comment (emitche) [inline app/views/admin/organizations/index.html.slim]: Allows us to sort by countries in the index.

File: app/views/admin/organizations/_form.html.slim [hunk 4]
```diff
@@ -0,0 +1,66 @@
+= simple_form_for(@coop,
```
Comment (emitche) [inline app/views/admin/organizations/_form.html.slim]: This block may need some consistent tabbing and spacing.

File: app/views/admin/organizations/_form.html.slim [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+        = content_tag(:option, country.name, value: country.id)
+
+  div.form-group.row
+    label.col-lg-3 Website
+    span.col-lg-6.view-element
+      - unless @coop.website.nil?
+        = link_to(@coop.website, @coop.website)
+    = f.input_field :website
+
+  div.form-group.row
```
Comment (emitche) [inline app/views/admin/organizations/_form.html.slim]: This isn't fully set up yet until we have people in database. And adding members not set up yet.

File: app/controllers/admin/organizations_controller.rb [hunk 6]
```diff
@@ -6,4 +6,27 @@ def index
...
+
+  def new
+    @coop = Organization.new
+    @countries = Country.all
+  end
+
+  def update
+    @coop = Organization.find(params[:id])
+    @coop.update!(organization_params)
+    # redirect_to admin_organization_url(@coop)
```
Comment (emitche) [inline app/controllers/admin/organizations_controller.rb]: Remove.

File: app/assets/javascripts/admin/forms/forms.js.coffee [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    $(form).attr('method', 'put')
+    data = $(".organization-record").serialize()
+    location = window.location.href
+
+    $.ajax
+      method: 'PUT',
+      url: location,
+      data: data,
+      success: (data) ->
+        console.log("Updated")
```
Comment (emitche) [inline app/assets/javascripts/admin/forms/forms.js.coffee]: Remove console log

File: app/assets/javascripts/admin/forms/forms.js.coffee [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+        window.location.href = data
+      error: (data) ->
+        console.log(data)
```
Comment (emitche) [inline app/assets/javascripts/admin/forms/forms.js.coffee]: Errors should be loading on server side with Simple Form and alerts. This console should not be here.

**Comment by emitche** [comment]
Discussed this with Fuzzy. Fuzzy to take this one on to polish up. Closing the pull request, but keeping the branch here.

**Comment by emitche** [comment]
Reopening so this work does not get lost.

**Comment by linesarefuzzy** [comment]
Still need to write tests. Oops.

**Comment by emitche** [comment]
Can this branch be deleted?

**Pull request merged**